transformers/events/log_item_update/entity.go

```diff
@@ -0,0 +1,9 @@
+package log_item_update
```

Comment:
Was _thinking_ of putting these Oasis transformers in their own `oasis` directory - curious what folks think about that.

Reply:
I like the idea! One thing I'd consider is if we might want to have the Oasis stuff run in a whole separate repo (`vdb-oasis-transformers`) and a different db schema. I think that'd be a cool way to start building off of the plugin architecture, but also may be too much complexity for the time being

Reply:
🤔 that's a really interesting idea. I wonder if its possible to start the transformers here, and then move them over to their own repo at some point. Or if that would be more trouble than it's worth.